libphobos/src/std/math.d

```diff
@@ -5065,7 +5131,32 @@ private:
             {
                 asm pure nothrow @nogc
                 {
-                    "fclex; fldcw %[cw]" : : [cw] "m" newState;
+                    "fclex; fldcw %0" : : "m" newState;
+                }
+
+                // Also update MXCSR, SSE's control register.
+                if (haveSSE)
+                {
+                    uint mxcsr;
+                    asm pure nothrow @nogc
+                    {
+                        "stmxcsr %0" : "=m" mxcsr;
+                    }
+
+                    /* In the FPU control register, rounding mode is in bits 10 and
+                       11. In MXCSR it's in bits 13 and 14. */
+                    mxcsr &= ~(ROUNDING_MASK << 3);             // delete old rounding mode
+                    mxcsr |= (newState & ROUNDING_MASK) << 3;   // write new rounding mode
```

Comment:
Why did upstream use enum/immutable here? Was that a manual optimization for DMD? ;-)
```d
enum ROUNDING_MASK_SSE = ROUNDING_MASK << 3;
immutable newRoundingModeSSE = (newState & ROUNDING_MASK) << 3;
```